eng/common/scripts/TypeSpec-Project-Sync.ps1

```diff
@@ -20,8 +20,12 @@ function AddSparseCheckoutPath([string]$subDirectory) {
 
 function CopySpecToProjectIfNeeded([string]$specCloneRoot, [string]$mainSpecDir, [string]$dest, [string[]]$specAdditionalSubDirectories) {
     $source = "$specCloneRoot/$mainSpecDir"
-    Copy-Item -Path $source -Destination $dest -Recurse -Force
     Write-Host "Copying spec from $source to $dest"
+    # $mainSpecDir is the PR folder, we just need to copy its subfolders which include the typespec project folder
+    Get-ChildItem –Path "$source" -Exclude @("data-plane", "resource-manager")|
```

Comment:
I think we can also sync these folders there isn't any real value in excluding them.

Reply:
Hello @weshaggard  `data-plane` and `resource-manager` is useless for generating SDK from typespec, and it is huge, so we skip to copy them to our template folder.

Reply:
I know they have specs but are they really that big? Do we have any real data that says they make it take too much time? It just feels like if we can eliminate this special case it might make things easier.

Reply:
If we decide to exclude them after all, I think adding a comment explaining why is that would be worthwhile. Would be nice if it would say how big is the slowdown.